lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
> My memory is hazy, but I think the logic in the UI that collapses multiple instances of duplication within a single file to just one rendered issue (with other locations attached) relies on the fact that all the instances are the same fingerprint.

@pbrisbin The UI doesn't do this - at least for the `issues > duplication` tab.

It shows all of the issues. 

e.g. https://codeclimate.com/repos/568aeda48c5d387d9200048d/issues/categories/duplication

in `flask/blueprints.py`